docs/deploy-ota.md

```diff
@@ -0,0 +1,62 @@
+# OTA Deployments
+
+## Overview
+
+![OTA Deployment](./img/ota-flow.png)
+
+## Internal Deployments
+
+Internal OTA deployments should be performed automatically upon all merges into main. In cases where the fingerprint diff results in incompatible native changes, a new client build will automatically be ran and deployed to TestFlight (iOS) or delivered in Slack (Android).
+
+## Production Deployments
+
+### Prerequisites
+
+- Remove any internal client from your device and download the client from the App Store/Google Play. This will help for testing as well as retrieving the build number.
+- You should have signed in to EAS locally through npx eas login. You will need to modify the build number in a subsequent step.
+- Identify the build number of the production app you want to deploy an update for. iOS and Android build numbers are divergent, so you will need to find both
```

Comment:
Nit: the build number is just the (239) or whatever, right?

Reply:
Oh yea, there's a photo that you can't see here that highlights the number you're looking for. But yes, that is correct.